### ISSUE TITLE
Factuality and Extraction confidence

### ISSUE THREAD

**Issue by bsharpataz** [clulab/eidos#407]
- export a richer/finer estimation of the reader's confidence in a given extraction

**Comment by bsharpataz** [comment]
subsumes #395 ?

**Comment by MihaiSurdeanu** [comment]
Yes, this does subsume #395. Negation and hedging should be used as features in the factuality classifier. Can you please close #395?
